src/static/partials/add-workshop.html

```diff
@@ -9,23 +9,28 @@
       <div class="col-md-4 inputGroupContainer">
 	<div class="input-group">
 	  <label><input type="radio" name="version" ng-model="version" value="online"/ required> online</label>&emsp;&emsp;
-	  <label><input type="radio" name="version" ng-model="version" value="offline"/ required> offline (college cloud edition)</label>
+	  <label><input type="radio" name="version" ng-model="version" disabled="" value="offline"/ required> offline (college cloud edition)</label>
 	  <span style="color:red" ng-show="myForm.version.$dirty && myForm.version.$invalid">
 	    <span ng-show="myForm.version.$error.required">Choose workshop version</span>
 	  </span>
 	</div>
       </div>
     </div>
-    
     <div class="form-group">
       <label class="col-md-4 control-label">Workshop Name *</label>  
       <div class="col-md-4 inputGroupContainer">
 	<div class="input-group">
-	  <input type="text" name="user" ng-model="name"  ng-pattern="/^[a-zA-Z0-9._-\s]*$/"  class="form-control"  placeholder="Workshop Name" required>
+	  <input type="text" type="number" ng-mouseover="targetWorkshopName('over')"
+               ng-mouseleave="targetWorkshopName('unover')"
+               name="workshops" min="0" ng-model="workshops"
```

Comment:
name and ng-model should be name="name" and ng-model="name" instead of
name="workshops"
ng-model="workshops"
The above changes will break the functionality of adding a workshop

Reply:
Fixed.
https://github.com/vlead/outreach-portal/commit/2b00dbfc509e84680e8829026d7bfcb720903342